build.sbt

```diff
@@ -374,6 +372,19 @@ lazy val codegenPlugin = (projectMatrix in file("modules/codegen-plugin"))
     scriptedBufferLog := false
   )
 
+lazy val decline = (projectMatrix in file("modules/decline"))
+  .settings(
+    name := "decline",
+    isCE3 := true,
+    libraryDependencies ++= List(
+      "org.typelevel" %%% "cats-core" % "2.7.0",
```

Comment:
One last thing: I see that we have things like `Dependencies.Weaver.cats.value` can you use them when possible?

I see that cats is there, and decline & decline effect could be moved in there (because decline is already in use by the cli)

Reply:
done